pepper-apis/studybuilder-cli/studies/osteo/emails/reconsent_email_verification_es.html

```diff
@@ -0,0 +1,120 @@
+             alt="Osteosarcoma Project logo" style="-ms-interpolation-mode: bicubic; border: 0; outline: none; text-decoration: none;"
+             width="230px" height="63px" border="0">
+      </a>
+    </td>
+  </tr>
+
+  <tr style="width: 540px;">
+    <td style="padding: 50px 0 0 0;">
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Estimado(a) {{participant_firstName}},</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Gracias por participar en el Proyecto sobre Osteosarcoma. Tu padre o madre nos proporcionó tu correo electrónico para poder informarte los próximos pasos del proceso.</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Como recordatorio, el objetivo del Proyecto sobre Osteosarcoma es transformar la atención oncológica al permitir que todos los pacientes a los que alguna vez se les haya diagnosticado osteosarcoma aceleren la investigación biomédica al compartir sus muestras de cáncer, información clínica y sus experiencias. Para lograr estos objetivos, los conjuntos de datos que contienen datos clínicos, genómicos, moleculares y notificados por los pacientes vinculados (sin incluir ninguna información personal) se compartirán periódicamente con la comunidad de investigación biomédica a través de repositorios científicos como el CBioPortal for Cancer Genomics y el Genomic Data Commons del Instituto Nacional del Cáncer (NCI).</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">En unos pocos meses, alcanzarás la edad conocida, en términos legales, como la mayoría de edad. En ese momento, te pediremos que firmes un nuevo formulario de consentimiento para seguir participando en OSproject.</p>
+
+      <p style="font-family: 'Source Sans Pro', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">Por ahora, haz clic a continuación para verificar tu dirección de correo electrónico. Te enviaremos un correo electrónico con más información sobre la renovación del consentimiento poco después de tu cumpleaños.</p>
+    </td>
+  </tr>
+  <tr style="height:0px">
+    <td style="border-left:solid #ffffff 1px;border-right:solid #ffffff 1px;border-bottom:solid #ffffff 1px;border-top:solid #ffffff 1px;vertical-align:top;background-color:#ffffff;padding:20px 0px 0px 0px">
+      <div>&nbsp; <a href="{{baseWebUrl}}/verify?invitation=-ddp.invitationId-&next=%2Fthank-you" style="font-size:16px;font-family:Helvetica,Arial,sans-serif;font-weight:normal;color:#ffffff;text-decoration:none;background-color:#7154ff;border-top:15px solid #7154ff;border-bottom:15px solid #7154ff;border-left:25px solid #7154ff;border-right:25px solid #7154ff;border-radius:3px;display:inline-block"
```

Comment:
This should be dynamic convention {{baseWebUrl}}
Please search for -ddp in *_es.html files for such stuff missed